src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -1636,9 +1593,8 @@ class MacroAssembler: public Assembler {
     load_pc_relative_instruction_size = 2 * instruction_size // auipc, ld
   };
 
-  enum NativeShortCall {
-    trampoline_size        = 3 * instruction_size + wordSize,
-    trampoline_data_offset = 3 * instruction_size
+  enum NativeFarCall {
+    trampoline_size        = 1 * instruction_size + wordSize,
```

Comment:
Call address stub size?

Reply:
I am a bit worried that `MacroAssembler::NativeShortCall` could be confusing with `MacroAssembler::far_call` in naming, so I simply removed this enum. I also improved the code comment about different types of calls. Let me know.